{
  TestRunner aTestRunner=new TestRunner();
  try {
    TestResult r=aTestRunner.start(args);
    if (!r.wasSuccessful())     System.exit(FAILURE_EXIT);
    System.exit(SUCCESS_EXIT);
  }
 catch (  Exception e) {
    System.err.println(e.getMessage());
    System.exit(EXCEPTION_EXIT);
  }
}
